updated expo android config docs (#3974)
braze-inc/braze-docs: _docs/_developer_guide/platform_integration_guides/react_native/push_notifications.md

Set the `enableBrazeIosPush` and `enableFirebaseCloudMessaging` props to enable push for iOS and Android, respectively.

## Android Setup

> Supply your `firebaseCloudMessagingSenderId`. See the [Android integration instructions]({{site.baseurl}}/developer_guide/platform_integration_guides/android/push_notifications/android/integration/standard_integration#step-4-set-your-firebase-credentials) on retrieving your sender ID. 

Reviewer: ```suggestion
1. Supply your `firebaseCloudMessagingSenderId`. See the [Android integration instructions]({{site.baseurl}}/developer_guide/platform_integration_guides/android/push_notifications/android/integration/standard_integration#step-4-set-your-firebase-credentials) on retrieving your sender ID. 
```

> Supply your `firebaseCloudMessagingSenderId`. See the [Android integration instructions]({{site.baseurl}}/developer_guide/platform_integration_guides/android/push_notifications/android/integration/standard_integration#step-4-set-your-firebase-credentials) on retrieving your sender ID. 
> 
> You will additionally have to add your `google-services.json` file to `assets/google-services.json` in your application's expo folder. This file is required when setting `enableFirebaseCloudMessaging: true` in your configuration.

Reviewer 2: ```suggestion

#### Step 1.1
Set the `firebaseCloudMessagingSenderId` config prop in your `app.json`. See the [Android integration instructions]({{site.baseurl}}/developer_guide/platform_integration_guides/android/push_notifications/android/integration/standard_integration#step-4-set-your-firebase-credentials) on retrieving your sender ID. 

#### Step 1.2
Add your `google-services.json` file to your application's`assets` folder. This file is required when setting `enableFirebaseCloudMessaging: true` in your configuration.
```

Reviewer 2: Thoughts on adding instructions on downloading the `google-services.json` file from Firebase? Or can we assume that people know how to do that?

Author: The "android integration instructions step 4" linked beforehand has those instructions.

> You will additionally have to add your `google-services.json` file to `assets/google-services.json` in your application's expo folder. This file is required when setting `enableFirebaseCloudMessaging: true` in your configuration.

Reviewer: ```suggestion
2. Add your `google-services.json` file to `assets/google-services.json` in your application's expo folder. This file is required when setting `enableFirebaseCloudMessaging: true` in your configuration.
```

Set the `enableBrazeIosPush` and `enableFirebaseCloudMessaging` props to enable push for iOS and Android, respectively.

> ## Android Setup

Reviewer 3: ```suggestion
### Android setup
```

Set the `enableBrazeIosPush` and `enableFirebaseCloudMessaging` props to enable push for iOS and Android, respectively.

> For Android, you will also need to supply your `firebaseCloudMessagingSenderId`. See the [Android integration instructions]({{site.baseurl}}/developer_guide/platform_integration_guides/android/push_notifications/android/integration/standard_integration#step-4-set-your-firebase-credentials) for instructions on retrieving your sender ID. You will additionally have to add your `google-services.json` file to `assets/google-services.json` in your application's expo folder. This file is required when setting `enableFirebaseCloudMessaging` to `true` in your configuration.

Reviewer: ```suggestion
## Android Setup
Supply your `firebaseCloudMessagingSenderId`. See the [Android integration instructions]({{site.baseurl}}/developer_guide/platform_integration_guides/android/push_notifications/android/integration/standard_integration#step-4-set-your-firebase-credentials) on retrieving your sender ID. 

You will additionally have to add your `google-services.json` file to `assets/google-services.json` in your application's expo folder. This file is required when setting `enableFirebaseCloudMessaging: true` in your configuration.
```
